applications/FluidDynamicsApplication/python_scripts/navier_stokes_embedded_ausas_solver.py

```diff
@@ -39,7 +39,7 @@ def __init__(self, main_model_part, custom_settings):
             "echo_level": 0,
             "time_order": 2,
             "compute_reactions": false,
-            "reform_dofs_at_each_step": true,
```

Comment:
i am not sure on this one. With the false it is faster. With the true it is "safer".

i still accept the PR, just consider my comment.